src/content/blog/web-framework-performance-report.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: "2023 Web Framework Performance Report"
```

Comment:
Should the URL match the fact that this is intended to be the **2023** Web Framework Performance report? Right now it's just `/blog/web-framework-performance-report`. I'd rename this slug to include `2023`